kobo/worker/logger.py

```diff
@@ -71,6 +71,10 @@ def stop(self):
 
 class LoggingIO(object):
     """StringIO wrapper that also writes all data to a logging thread."""
+    __slots__ = (
```

Comment:
I have covscan sources, is there a practical way for me to reproduce the problem?  I'd at least like to have a try finding a plausible explanation for this before merging.